besu/src/test/java/org/hyperledger/besu/cli/BesuCommandTest.java

```diff
@@ -1829,7 +1829,7 @@ public void syncMode_invalid() {
     assertThat(commandOutput.toString(UTF_8)).isEmpty();
     assertThat(commandErrorOutput.toString(UTF_8))
         .contains(
-            "Invalid value for option '--sync-mode': expected one of [FULL, FAST, X_SNAP, X_CHECKPOINT] (case-insensitive) but was 'bogus'");
+            "Invalid value for option '--sync-mode': expected one of [FULL, FAST, SNAP, CHECKPOINT, X_SNAP, X_CHECKPOINT] (case-insensitive) but was 'bogus'");
```

Comment:
I don't love this but it was not easy to remove so I figure we can live with it until 24.4.0